tests/attacks/membership_inference/label_only_boundry_distance_test.py

```diff
@@ -5,13 +5,19 @@
 from privacy_evaluator.datasets.cifar10 import CIFAR10
 from privacy_evaluator.classifiers.classifier import Classifier
 
+import torch.nn as nn
+
 
 def test_inference_label_only_decision_boundary_attack():
```

Comment:
What is the purpose of this test? It does not seem really meaningful to me.

Reply:
At the moment, we are just testing if no exceptions are raised and everything is running. I tried to implement a more meaningful test for the output of the attack, but this is not so simple. The results of the attack change once in a while (probably due to different used seeds in ART). Therefore testing specific values is not possible. Furthermore, for this specific attack, the fitting process takes too long for more samples, thus we limited it to just one sample, which is definitely not the way it should be used. I added an additional assertion test to the  `MembershipInferenceBlackBoxRuleBasedAttack` and `MembershipInferenceBlackBoxAttack` attack but for this one, I am not sure what we can do here.